models/domains_tlds.php

```diff
@@ -369,7 +365,7 @@ public function edit($tld, array $vars)
                 'descriptions' => $package->descriptions,
                 'module_row' => isset($vars['module_row'])
                     ? $vars['module_row']
-                    : (isset($package->module_row) ? $package->module_row : null),
+                    : null,
```

Comment:
Looks great!  This is this only thing I'm wondering about?  Why did we change this defaulting behavior?

Reply:
Previously if $vars['module_row'] was not provided, it defaulted back to $package->module_row, but if a module does not have any module_row (e.g. a recently installed module), $package->module_row returned 0, and 0 is an invalid value and therefore $this->Packages->edit() returned an error that the module_row_id is invalid, preventing the TLD from being updated.

Reply:
I think I could use `(isset($package->module_row) && $package->module_row > 0 ? $package->module_row : null),` 🤔 instead, to cover all scenarios.